Sources/Identity/CustomerInfoManager.swift

```diff
@@ -375,6 +381,21 @@ private extension CustomerInfoManager {
         }
     }
 
+    private func cachedOrRequestCustomerInfoFetcher(
```

Comment:
I've been thinking, shouldn't this also try to request customer info first if cache is stale, instead of relying directly on the cache? 

I think this matters mostly In the event that all purchases have already been posted, this will just pick the cached version, which might be stale.

Reply:
Good point, need to check if cache is stale. But actually this depends on the fetch policy.
TODO: need to make this depend on `CacheFetchPolicy`.

Reply:
Solution:
Replace `customerInfoFetcher` with a call to the top-level `customerInfo(fetchPolicy:)` but with a flag preventing it from post any unfinished transactions anymore.

Reply:
> but with a flag preventing it from post any unfinished transactions anymore.

I did that in `customer-info-manager-optional-post-transactions`, but I think we can't do that.
I'm going to update this with a comment explaining why.